src/routes/choose-outfit/+page.svelte

```diff
@@ -52,115 +62,97 @@
-				>Return to Preferences</Button
-			>
-			<Button on:click={() => submitOutfitChoice()}>Submit Outfit Choice</Button>
-		</div>
-	{:else}
-		<Header type="h1">Selected Outfit:</Header>
-		<div class="display_outfit">
-			<Header type="h2">{filteredOutfits[outfitCount].name}</Header>
-			<div class="img_wrapper">
-				<img
-					src={clothesStore.getClothingItemById(filteredOutfits[outfitCount].topid).img}
-					alt="top"
-				/>
-				<img
-					src={clothesStore.getClothingItemById(filteredOutfits[outfitCount].bottomid).img}
-					alt="bottom"
-				/>
-			</div>
-		</div>
-	{/if}
+	<Tabs activeTab={activeTabNumber} bind:this={tabs}>
+		<svelte:fragment slot="labels">
+			<TabLabel tabnum="1">Choose Outfit</TabLabel>
+			<TabLabel tabnum="2">Selected Outfit</TabLabel>
+		</svelte:fragment>
+
+		<svelte:fragment slot="contents">
+			<TabContent tabnum="1">
+				<StandardContentLayout>
+					<div class="box">
+						<Header type="h2">What type of outfit are you looking for today?</Header>
+						<div class="split_container">
+							<div class="popover_content">
+								<Popover on:popoverItemsChanged={handlePopoverItemsChanged}>
+									<PopoverChipTrigger class="popover_content" slot="trigger" label="Preferences" />
+									<PopoverMultiSelectContent
+										class="popover_content"
+										slot="content"
+										items={preferenceItems}
+									/>
+								</Popover>
+							</div>
+						</div>
+					</div>
+					<div class="outfit_cards">
+						{#each $outfitStore as outfit}
+							{#if filteredOutfits.includes(outfit)}
+								<div class="display_outfit">
+									<Header type="subheader">{outfit.name}</Header>
+									<div class="img_wrapper">
+										<img src={clothesStore.getClothingItemById(outfit.topid).img} alt="top" />
+										<img src={clothesStore.getClothingItemById(outfit.bottomid).img} alt="bottom" />
+									</div>
+									<Button on:click={() => submitOutfitChoice(outfit.id)}>Choose this Outfit</Button>
+								</div>
+							{/if}
+						{/each}
+					</div>
+				</StandardContentLayout>
+			</TabContent>
+
+			<TabContent tabnum="2">
+				{#if $selectedOutfitId != undefined}
+					<StandardContentLayout>
+						<Header type="h1">Selected Outfit</Header>
+						<div class="display_outfit full">
+							<Header type="h2">{outfitStore.getOutfitById($selectedOutfitId).name}</Header>
+							<div class="img_wrapper">
+								<img
+									src={clothesStore.getClothingItemById(
+										outfitStore.getOutfitById($selectedOutfitId).topid
+									).img}
+									alt="top"
+								/>
+								<img
+									src={clothesStore.getClothingItemById(
+										outfitStore.getOutfitById($selectedOutfitId).bottomid
+									).img}
+									alt="bottom"
+								/>
+							</div>
+						</div>
+					</StandardContentLayout>
+				{/if}
+			</TabContent>
+		</svelte:fragment>
+	</Tabs>
+	<Toast
+		title="Incorrect Preferences"
```

Comment:
The wording of this doesn't seem quite right to me, since the preferences aren't really "incorrect" in any way, but there just aren't any outfits that match those preferences. Maybe it could just say "No outfits match your selected preferences"

Reply:
I updated this